### ISSUE TITLE
SickGear <-> nzbToMedia <-> SABnzbd show name with year issue

### ISSUE THREAD

**Issue by ouija** [SickGear/SickGear#1370]
#### Branch version commit hash: 043fe387e222aadea3ead5e7e50b53883b77c4f4

This may or may not be a direct result of SickGear but rather may be a configuration issue or something else, but figured I'd pose the question in case you have any insight.

I recently switched to SickGear, and have it working beautifully with SABnzbd and nzbToMedia scripts -- However, there is one thing that I'm trying to resolve, which is minor but somewhat annoying:

I've found that completed downloads don't seem to be moving into the "Show Name" directory (defined in SickGear) when processing is completed,  unless I have **TV Sorting** enabled in SABnzbd -- which is fine and seems to resolve this, _however_, this can cause isssues if there are different shows with the same name and the year is then added ti the show name.

For example, let's say a show is called "Example (2021)", and this is the folder name that SickGear created when adding the show and configured to use.   When a download is completed and processed _(using nzbToMedia)_, SABnzbd seems to be creating a folder instead called "Example 2021" _(without brackets around the year)_ and placing the download within this folder, instead of the one defined in SickGear.

If I disable **TV Sorting** within SABnzbd, then this download is placed under a folder based on the downloaded file name, and not the folder defined for the show in SickGear.

It's not a major thing, because I've been renaming folders and re-mapping them in SickGear, but where it is a bit of a pain is if you are adding new shows in SickGear and if the show name has a year in it, and I figured I'd ask in case it's something as simple as an incorrect configuration problem.

Thanks!

**Comment by JackDandy** [comment]
Hi

SickGear natively performs complete media process folder handling and renaming; rename features should not be enabled in clients. 

SickGear supports and maintains scripts that are used between SickGear and clients; these are located in the Post installation section [here](https://github.com/SickGear/SickGear/wiki/Installation-Instructions#post-installation).

I also recommend you take a look at the FAQ page to learn from the carefully maintained help written from user feedback. The texts show how to set up SickGear to run without any of the issues you write about. 

There is not an actual issue here, therefore, as per our normal handling, this thread will now be marked invalid. Feel free to ask anything else below, even though it is closed that's no problem.

Good luck.

**Comment by ouija** [comment]
Right -- all good!  Appreciate the insights and I just tested out using the processing scripts included for SickGear [and they do resolve this year naming issue] but I realized now  _why_ I had things configured the way I did -- as I sometimes just download "one-off" episodes, and SickGear won't process them unless they exist in the show database (or alt title)  and sometimes I just want to download one or two episode of a certain show _without_ adding it to SickGear but still need it to be processed/renamed.

Consider this resolved, unless you have any solutions to what I've described!  Thanks again.
